.github/workflows/spellcheck.yml

```
@@ -10,4 +10,4 @@ jobs:
         with:
           python-version: '3.10'
       - run: pip install codespell
-      - run: codespell -S .git,docson,locale,currency.csv -L tne,zar .
+      - run: codespell -S .git,docson,locale,currency.csv -L tne,zar,fo,nam .
```

We can exclude external codelists (country.csv has "Viet Nam").
```suggestion
      - run: codespell -S .git,docson,locale,country.csv,currency.csv -L fo,tne,zar .
```